ci-operator/config/shiftstack/ci/shiftstack-ci-release-4.17__techpreview.yaml

```diff
@@ -44,6 +44,7 @@ tests:
   steps:
     cluster_profile: openstack-vexxhost
     env:
+      ETCD_DISK_SPEED: slow
```

Comment:
please enable it in more jobs than just this one. We really want to see if that makes a difference. If possible jobs from multiple cluster profiles (CSI jobs, etc).

Reply:
I agree. I didn't want to make this PR bigger, but I prefer to test it with more jobs.